netkan/netkan/indexer.py

```diff
@@ -129,7 +129,7 @@ def status_attrs(self, new=False):
         }
         resources = getattr(self.ckan, 'resources', None)
         if resources:
-            attrs.resources = resources
+            attrs['resources'] = resources
         if new:
```

Comment:
Heh, dictionary vs attribute, still catches me sometimes. I actually wanted to create an object for this, but the Model class seem to require a dict.